Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -172,29 +172,35 @@ private void ChangeDamage(EntityUid uid, TemperatureComponent temperature)
             var y = temperature.DamageCap;
             var c = y * 2;
 
-            if (temperature.CurrentTemperature >= temperature.HeatDamageThreshold)
+            var ev = new ChangeDamageGetThresholdsEvent();
+            RaiseLocalEvent(uid, ref ev);
+
+            var heatDamageThreshold = ev.HeatDamageThreshold ?? temperature.HeatDamageThreshold;
+            var coldDamageThreshold = ev.ColdDamageThreshold ?? temperature.ColdDamageThreshold;
+
+            if (temperature.CurrentTemperature >= heatDamageThreshold)
```

Comment:
Instead of doing it specifically for the cryo pods, I added a new ContainerTemperatureDamageThresholdsComponent to allow a parent container to override the temperature thresholds of a contained entity by simply adding the component in YAML. Not sure if it's overengineering but I think there might be more use cases for this and it isn't really expensive or anything, I think.